mobile/src/main/res/values-de/strings.xml

```diff
@@ -1,9 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources xmlns:tools="http://schemas.android.com/tools" tools:ignore="MissingTranslation">
     <!-- General app strings -->
+
```

Comment:
why adding so much empty lines? :/

Reply:
empty lines are placeholders for not translatable values. It's easier to compare the default and the german strings.xml